packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -361,6 +361,9 @@ export class Loan {
    * @param {string} metadata.vaultId Vault id
    * @param {string| string[]} metadata.amounts In "amount@symbol" format
```

Comment:
```suggestion
   * @param {string| string[]} [metadata.amounts] In "amount@symbol" format
```

Reply:
i thought this from the actual RPC perspective, then it's optional. But this is better since it also enforces the restrictions of not using `amounts` and `loans` together.